src/AspNet.Security.OAuth.GitLab/GitLabAuthenticationDefaults.cs

```diff
@@ -0,0 +1,56 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+
+namespace AspNet.Security.OAuth.GitLab
+{
+    /// <summary>
+    /// Default values for GitLab authentication
```

Comment:
Please follow convention and change this to `Default values used by the GitLab authentication middleware.`

Reply:
Fixed